src/components/Envelope.vue

```diff
@@ -635,6 +635,15 @@ export default {
 .list-item-style.selected {
 	background-color: var(--color-background-dark);
 }
+.list-item-style {
+	.draft {
+		line-height: 130%;
+
+		em {
+			font-style: italic;
```

Comment:
We do not use italic anywhere in the interface for legibility, shouldn’t do it here either. :) As per [design guidelines](https://docs.nextcloud.com/server/latest/developer_manual/design/foundations.html#typography) @ChristophWurst